torch/csrc/distributed/c10d/ProcessGroup.cpp

```diff
@@ -50,7 +50,8 @@ std::string opTypeToString(OpType opType) {
   return "UNKNOWN";
 }
 
-bool isP2POp(OpType opType) {
+bool isP2POp(OpType opType, bool batchP2P /*= false*/) {
+  if (batchP2P) return false;
```

Comment:
Add a comment indicating that this basically means only dist.send/isend, dist.recv/irecv are the only ops considered p2p, and batch_isend_irecv is not considered p2p and we treat it more as a collective?

Reply:
That's right. Will do!